client/stylesheets/shared/_embed.scss

```diff
@@ -50,6 +50,7 @@
 
 	#screen-meta-links {
 		position: relative;
+		height: $header-height + 50px;
```

Comment:
Maybe we could use a larger `margin-bottom` instead of an explicit height.  I'm always cautious of setting a height and creating an overflow issue if the contents change.

Reply:
Thank you @joshuatf. Let me play with margin-bottom and see if that works 👍

Reply:
@joshuatf used margin-bottom instead in https://github.com/woocommerce/woocommerce-admin/pull/6118/commits/003454356b6440637cd660689f4785c232e81192.

For the layout issue with the new navigation enabled, I think that's an existing issue. This is happening in the main branch as well. I will create a new ticket to address it.

![Screen Shot 2021-01-21 at 3 26 11 PM](https://user-images.githubusercontent.com/4723145/105424933-288b3f80-5bfd-11eb-9a97-53ccb0c63590.jpg)
![Screen Shot 2021-01-21 at 3 26 06 PM](https://user-images.githubusercontent.com/4723145/105424936-2a550300-5bfd-11eb-8023-6f76559bf2af.jpg)